includes/settings/settings-ppec.php

```diff
@@ -324,6 +324,13 @@
 		'desc_tip'    => true,
 		'description' => __( 'This enables PayPal Credit, which displays a PayPal Credit button next to the Express Checkout button. PayPal Express Checkout lets you give customers access to financing through PayPal Credit® - at no additional cost to you. You get paid up front, even though customers have more time to pay. A pre-integrated payment button shows up next to the PayPal Button, and lets customers pay quickly with PayPal Credit®.', 'woocommerce-gateway-paypal-express-checkout' ),
 	),
+	'checkout_product' => array(
```

Comment:
can we make a better setting's name here? I don't mind with long name like `checkout_on_single_product_enabled` or something like that.